core/src/main/java/org/apache/accumulo/core/client/Connector.java

```diff
@@ -55,38 +56,6 @@
   public abstract BatchScanner createBatchScanner(String tableName, Authorizations authorizations,
       int numQueryThreads) throws TableNotFoundException;
 
-  /**
-   * Factory method to create a BatchDeleter connected to Accumulo.
```

Comment:
Connector is deprecated and the AccumuloClient interface doesn't have these methods.  They probably don't need to be removed.

Reply:
Restored.